pkg/openapi/server.spec.yaml

```diff
@@ -2448,6 +2420,11 @@ components:
           $ref: '#/components/schemas/storageV2Spec'
         status:
           $ref: '#/components/schemas/storageV2Status'
+    storagesV2List:
```

Comment:
Too many plurals? "storages" then "List" suggests it's a list of lists of storage; I think you'd want "storageV2List" here.